src/locales/index.ts

```diff
@@ -1,8 +1,10 @@
 import { en } from './en'
+import { ca_ES } from './ca_ES'
 
 // If you wish to define additional locales, add them here separated by `|`
-export type DefinedLocales = 'en'
+export type DefinedLocales = 'en | ca_ES'
```

Comment:
This should be a TypeScript union of multiple strings like:
```suggestion
export type DefinedLocales = 'en' | 'ca_ES'
```